src/bin/download.rs

```diff
@@ -56,21 +56,27 @@ fn main() {
         exit_with_status(1, &tmp_file_path);
     }
 
-    println!("Downloading input via aoc-cli...");
 
-    let mut cmd_args = vec![
-        "download".into(),
+    let mut tmp_cmd_args = vec![
```

Comment:
What do you think about changing the order here a bit to get rid of this assignment?

```rs
    let mut cmd_args = vec![];

    if let Some(year) = args.year {
        cmd_args.push("--year".into());
        cmd_args.push(year.to_string());
    }

    cmd_args.append(&mut vec![
        "--file".into(),
        tmp_file_path.to_string_lossy().to_string(),
        "--day".into(),
        args.day.to_string(),
        "download".into(),
    ]);
```

Reply:
👍